app/src/main/java/eu/kanade/tachiyomi/data/sync/service/GoogleDriveSyncService.kt

```diff
@@ -151,6 +152,7 @@ class GoogleDriveSyncService(context: Context, json: Json, syncPreferences: Sync
 
 class GoogleDriveService(private val context: Context) {
     var googleDriveService: Drive? = null
+    private val redirectUri = "eu.kanade.google.oauth:/oauth2redirect"
```

Comment:
Use proper constant that way it's optimized.
```suggestion
        companion object {
        const val REDIRECT_URI = "eu.kanade.google.oauth:/oauth2redirect"
    }
```